doc/installation_raspbian.md

```diff
@@ -46,6 +46,7 @@ $ cd ~
 $ git clone https://github.com/IntelRealSense/librealsense.git
 $ cd librealsense
 $ sudo cp config/99-realsense-libusb.rules /etc/udev/rules.d/ 
+$ sudo cp config/99-realsense-d4xx-mipi-dfu.rules /etc/udev/rules.d/
```

Comment:
why change in raspbian? isn't it for rasberry pi only?

Reply:
followed up 99-realsense-libusb.rules

Reply:
What does it mean?
RPI does not support D457/Mipi device so why should the user copy that rule?

Just want to understand the change :)

Reply:
removed that line